### ISSUE TITLE
Darken the check box outline for better visibility

### ISSUE THREAD

**Issue by aurooba** [aurooba/wpaudit.site#20]
<img width="239" alt="Screen Shot 2022-03-15 at 10 54 42 AM" src="https://user-images.githubusercontent.com/6925260/158430539-a9f47b4d-29de-439c-9544-4b4085068886.png">

Currently the checkbox outline is not visible enough and fails the color contrast check. This needs to be remedied. Suggestion is to change it to the teal colour (#057B85).

**Comment by aurooba** [comment]
Fixed by [2e609ce](https://github.com/aurooba/wpaudit.site/commit/2e609cecccee7b1fb423176221ac06dde618eadb)
